docs/UserGuide.md

```diff
@@ -110,18 +123,22 @@ Examples:
 
 ### Removing specific field in person: `remove`
 
-Edits a specific field of a person in the address book, accessed by View method.<br>
+Removes a specific field of a person in the address book at the specified index. Can only be used in View Mode.<br>
 It requires one field:
 - h/: To be followed by INDEX of homework to be removed
 - a/: To be followed by INDEX of attendance to be removed
 - g/: To be followed by INDEX of grade to be removed
 - s/: To be followed by INDEX of session to be removed
 
+Allows for edits of multiple fields in a single command. <br>
+
 Format: `remove [h/ a/ g/ s/]INDEX`
 
 Examples:
 * `view 1` returns person in first index <br>
   `remove h/2` removes person at first index's *homework at second index*
+* `view 1` returns person in first index <br>
+  `remove h/2 a/1` removes person at first index's *homework at second index* and *attendance at first index*
```

Comment:
I think remove only works with one field. like `remove h/2` or `remove a/1` not both `remove h/2 a/1`

Reply:
![image](https://user-images.githubusercontent.com/89232083/198185773-99f315bc-ab03-434e-9163-35c0df188a1b.png)

![image](https://user-images.githubusercontent.com/89232083/198185799-95ecdb79-f820-4ad6-b042-464780f80fe8.png)

hihi but here like works thoooo

Reply:
Oops, sorry. I tested out like `remove a/1 a/2` and it didn't work for that. Thanks for letting me know :)

Reply:
hm mayb i shld say like allow for one remove for one field of the person

Reply:
yeap yeap that would be more clear too!